src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -60,262 +90,142 @@ def create_connectedk8s(cmd, client, resource_group_name, cluster_name,
             if location.lower() not in rp_locations:
                 raise CLIError("The connected cluster resource creation is supported only in the following locations: " + ', '.join(map(str, rp_locations)) + ". Please use the --location flag to specify right location.")
             break
+
+    # Check Release Existance
+    release_namespace = get_release_namespace(kube_config, kube_context) 
+    if release_namespace is not None:
+        # Loading config map
+        api_instance = kube_client.CoreV1Api(kube_client.ApiClient(configuration))
+        try:
+            configmap = api_instance.read_namespaced_config_map('azure-clusterconfig', 'azure-arc')
+        except ApiException as e:
+            raise CLIError("Exception when calling CoreV1Api->read_namespaced_config_map: %s\n" % e)
+        configmap_resource_group_name = configmap.data["AZURE_RESOURCE_GROUP"]
+        configmap_cluster_name = configmap.data["AZURE_RESOURCE_NAME"]
+        if connected_cluster_exists(client, configmap_resource_group_name, configmap_cluster_name):
+            if (configmap_resource_group_name.lower() == resource_group_name.lower() and configmap_cluster_name.lower() == cluster_name.lower()):
+                # Re-put connected cluster
+                public_key = client.get(configmap_resource_group_name, configmap_cluster_name).agent_public_key_certificate
+                cc = generate_request_payload(configuration, location, public_key, tags)
+                try:
+                    return sdk_no_wait(no_wait, client.create, resource_group_name=resource_group_name, cluster_name=cluster_name, connected_cluster=cc)
```

Comment:
In Re-PUT scenario do we need to install agents ?.

Reply:
Not required as we are checking things from config map which is part of the helm chart.